styles.css

```diff
@@ -343,4 +343,64 @@ body {
   .my-skills li img {
     object-fit: contain;
   }
+
+  .contact {
+    background-image: url("assets/contact-bg-image.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    align-items: center;
+    padding: 5rem 1.25rem;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+    gap: 2rem;
+  }
+
+  .contact h2 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    color: #fff;
+    margin-right: 1.25rem;
+  }
+
+  .contact p {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.25rem;
+    line-height: 1.75rem;
+    color: #fff;
+  }
```

Comment:
- [ ] `Paragraph` in the contact me section is centered according to the Figma design. Kindly fix this issue by adding: `text-align: center`.